src/Microsoft.ML.Data/StaticPipe/StaticPipeUtils.cs

```diff
@@ -10,7 +10,7 @@
 using Microsoft.ML.Runtime;
 using Microsoft.ML.Runtime.Data;
 
-namespace Microsoft.ML.Data.StaticPipe.Runtime
+namespace Microsoft.ML.StaticPipe.Runtime
```

Comment:
StaticPipeExtensions lives in Microsoft.ML.StaticPipe,
StaticPipeInternalUtils lives in Microsoft.ML.StaticPipe.Runtime
StaticPipeUtils lives in Microsoft.ML.StaticPipe.Runtime
I would expect StaticPipeUtils to live in same place as StaticPipeExtensions.
But what do I know.

Reply:
Things we expect *users* of pipelines to see/use is in `Microsoft.ML.StaticPipe`, things related to internal implementations or things used by components (which includes these `Utils`) is in the runtime. But if you have another scheme I'd be happy to consider it I suppose.